src/models/github-server-app.ts

```diff
@@ -48,6 +48,29 @@ export class GitHubServerApp extends Model {
 		});
 	}
 
+	/**
+	 * Get all GitHubServerApps for gitHubBaseUrl with installationId
+	 *
+	 * @param {{gitHubBaseUrl: string, installationId: number}}
```

Comment:
The parameters are 2 different inputs and not a single object. So better to separate them to avoid confusion.
```suggestion
   * @param gitHubBaseUrl
   * @param installationId
```